core/include/crypto/crypto.h

```diff
@@ -83,6 +83,22 @@ void crypto_authenc_copy_state(void *dst_ctx, void *src_ctx);
 
 #if defined(CFG_NXP_SE05X)
```

Comment:
Would it make sense to move this entire block to a separate .h file, `<crypto/nxp_se05x.h>`? It seems that normal crypto users will not use these function so this only adds noise to ignore. A comment or two wouldn't hurt either.

Reply:
@jenswi-linaro you mean temporarily until support for another APDU controlled secure element is added to OP-TEE?

Reply:
both of these interfaces are generic but the se05x is the only user currently

[SCP03](:https://www.cardlogix.com/glossary/scp03-secure-channel-protocol-3/#:~:text=SCP03%20is%20a%20Asymmetric%20Secure,resistant%20to%20overhearing%20and%20tampering.&text=SCP03%20uses%20the%20generic%20composition,CPA%20and%20INT%2DCTXT%20secure.)

[APDU](https://en.wikipedia.org/wiki/ISO/IEC_7816#7816-8:_Commands_and_mechanisms_for_security_operations)

Reply:
`crypto/apdu_se.h` is fine if you prefer that.

Reply:
ok

Reply:
I changed it for crypto/secure_element.h..hope that is ok